### PR TITLE
feat(Onboarding): Added video tutorial to empty meetings dash

### LINKED ISSUE #6652: Designed meeting view with demo and video
## Issue - Enhancement
Update the learning hub design for #6648 to also have a demo (#6649 ) so we can test both options at the same time

### Acceptance Criteria (optional)
Users can:
   - either go to a demo or watch a short tutorial video

This issue blocks #6648 implementation

### PR DESCRIPTION
**Pull request by Dschoordsch**
# Description

Fixes #6652 
Add a video tutorial link to the empty meetings dash experience.

TODO: Final links are missing

## Demo
meeting dash demo

## Testing scenarios

Open meeting dash without any active meetings, click the tutorial card.

## Final checklist

- [ ] I checked the [code review guidelines](../docs/codeReview.md)
- [ ] I have performed a self-review of my code, the same way I'd do it for any other team member
- [ ] I have tested all cases I listed in the testing scenarios and I haven't found any issues or regressions
- [ ] Whenever I took a non-obvious choice I added a comment explaining why I did it this way
- [ ] I added the label `One Review Required` if the PR introduces only minor changes, does not contain any architectural changes or does not introduce any new patterns and I think one review is sufficient'
- [ ] PR title is human readable and could be used in changelog

### CODE REVIEW

**Comment by Dschoordsch** [comment]
@igorlesnenko although I don't have the final video links, let's do the review already so maybe we can still make this ship.

File: packages/client/components/TutorialMeetingCard.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+  const maybeTabletPlus = useBreakpoint(Breakpoint.FUZZY_TABLET)
+  const {togglePortal: toggleModal, modalPortal} = useModal()
+
+  return (
+    <>
+      <CardWrapper maybeTabletPlus={maybeTabletPlus} onClick={toggleModal}>
+        <MeetingImgWrapper>
+          <MeetingImgBackground />
+          <MeetingTypeLabel>Tutorial</MeetingTypeLabel>
+          <MeetingImg src={THUMBNAIL} alt='' />
```
Comment (igorlesnenko) [inline packages/client/components/TutorialMeetingCard.tsx:110]: is empty alt expected?

**Review by igorlesnenko — APPROVED** [review]

File: packages/client/components/TutorialMeetingCard.tsx (hunk 1 above)
Comment (Dschoordsch) [reply packages/client/components/TutorialMeetingCard.tsx:110]: I think it falls into the [decorative image with adjacent text](https://www.w3.org/WAI/tutorials/images/decorative/#example-3-image-with-adjacent-text-alternative) category, so an empty alt is fine.

**Comment by Dschoordsch** [comment]
@igorlesnenko can you give this another quick look? I added segment events and exchanged the URLs.

**Review by igorlesnenko — APPROVED** [review]
It works!
image

Is it possible to somehow track if youtube video was actually played?

**Pull request merged**